barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_keccak_flavor.hpp

```diff
@@ -19,370 +19,13 @@
 #include "barretenberg/relations/relation_parameters.hpp"
 #include "barretenberg/relations/ultra_arithmetic_relation.hpp"
 #include "barretenberg/stdlib_circuit_builders/ultra_circuit_builder.hpp"
+#include "barretenberg/stdlib_circuit_builders/ultra_flavor.hpp"
 #include "barretenberg/transcript/transcript.hpp"
 
 namespace bb {
```

Comment:
curious: was there any reason why UltraKeccak wasn't a child of Ultra before? it's nice imo that you made this change, because the differences are much clearer

Reply:
not really other than that they were out of sync at some point  and it was faster to duplicate I guess